web/templates/user_profile_modal.hbs

```diff
@@ -3,9 +3,13 @@
         <div class="modal__container new-style" role="dialog" aria-modal="true" aria-labelledby="dialog_title">
             <div class="modal__header">
                 {{#unless is_bot}}
-                <div class="tippy-zulip-tooltip" data-tippy-content="{{last_seen}}">
-                    <span class="{{user_circle_class}} user_circle user_profile_presence"></span>
-                </div>
+                    {{#if is_active}}
+                        <div class="tippy-zulip-tooltip" data-tippy-content="{{last_seen}}">
+                            <span class="{{user_circle_class}} user_circle user_profile_presence"></span>
+                        </div>
+                    {{else}}
+                        <i class="fa fa-ban deactivated-user-icon tippy-zulip-tooltip" data-tippy-content="Deactivated user"></i>
```

Comment:
What is the function of the `deactivated-user-icon` here?

Reply:
Oh, it was a mistakenly left there. I used that for development purposes. I have removed that now.